docs/reST/ref/sprite.rst

```diff
@@ -19,7 +19,7 @@ of objects in the game. There is also a base Group class that simply stores
 sprites. A game could create new types of Group classes that operate on
 specially customized Sprite instances they contain.
 
-The basic Sprite class can draw the Sprites it contains to a Surface. The
+The basic Group class can draw the Sprites it contains to a Surface. The
 ``Group.draw()`` method requires that each Sprite have a ``Surface.image``
 attribute and a ``Surface.rect``. The ``Group.clear()`` method requires these
```

Comment:
I would also change
> requires that each Sprite have a `Surface.image` attribute and a `Surface.rect`. 

to
> requires that each Sprite has an `image` and a `rect` attribute.

because what's written there currently doesn't make much sense.